### ISSUE TITLE
npm and yarn both can't find serve due to an added /-/ path

### ISSUE THREAD

**Issue by neuroball** [zeit/serve#399]
Hello,

Both npm and yarn have issues installing serve, as the path is wrong at https://registry.yarnpkg.com/serve/ and at https://registry.npmjs.org/serve.

The path should be:
`https://registry.npmjs.org/serve/serve-6.5.8.tgz`

but in reality the path is: 
`https://registry.npmjs.org/serve/-/serve-6.5.8.tgz` with and added `/-/`.

Please let me know if this is the right place to report this issue, or if I need to directly report this to npm/yarn.

Cheers,
/Oliver/

**Comment by LujunWeng** [comment]
Hi,

It seems like some versions are missing, because I also got `Not Found` when trying to download serve-6.5.6 from `https://registry.yarnpkg.com/serve/-/serve-6.5.6.tgz`. It is not caused by the path `/-/`, I think. I changed it to 6.4.0, and it worked. And if I checked `yarn.lock` in a repo, all paths contain `/-/`. Actually, now the `https://registry.yarnpkg.com/serve/-/serve-6.5.6.tgz` works and yours as well. 

Cheers
